src/app/decorations/qgsdecorationtitledialog.cpp

```diff
@@ -0,0 +1,136 @@
+/***************************************************************************
+  qgsdecorationtitledialog.cpp
+  --------------------------------------
+  Date                 : November 2018
+  Copyright            : (C) 2018 by Mathieu Pellerin
+  Email                : nirvn dot asia at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsdecorationtitledialog.h"
+#include "qgsdecorationtitle.h"
+
+#include "qgisapp.h"
+#include "qgsexpression.h"
+#include "qgsexpressionbuilderdialog.h"
+#include "qgsexpressioncontext.h"
+#include "qgshelp.h"
+#include "qgsmapcanvas.h"
+#include "qgssettings.h"
+
+#include <QColorDialog>
+#include <QColor>
+#include <QFont>
+#include <QDialogButtonBox>
+#include <QPushButton>
+
+QgsDecorationTitleDialog::QgsDecorationTitleDialog( QgsDecorationTitle &deco, QWidget *parent )
+  : QDialog( parent )
+  , mDeco( deco )
+{
+  setupUi( this );
+  connect( buttonBox, &QDialogButtonBox::accepted, this, &QgsDecorationTitleDialog::buttonBox_accepted );
+  connect( buttonBox, &QDialogButtonBox::rejected, this, &QgsDecorationTitleDialog::buttonBox_rejected );
+  connect( mInsertExpressionButton, &QPushButton::clicked, this, &QgsDecorationTitleDialog::mInsertExpressionButton_clicked );
+
+  QgsSettings settings;
+  restoreGeometry( settings.value( QStringLiteral( "Windows/DecorationTitle/geometry" ) ).toByteArray() );
+
+  QPushButton *applyButton = buttonBox->button( QDialogButtonBox::Apply );
+  connect( applyButton, &QAbstractButton::clicked, this, &QgsDecorationTitleDialog::apply );
+
+  grpEnable->setChecked( mDeco.enabled() );
+
+  // label text
+  txtTitleText->setAcceptRichText( false );
+  if ( !mDeco.enabled() && mDeco.mLabelText.isEmpty() )
+  {
+    QString defaultString = QStringLiteral( "%1" ).arg( QgsProject::instance()->metadata().title() );
+    txtTitleText->setPlainText( defaultString );
+  }
+  else
+  {
+    txtTitleText->setPlainText( mDeco.mLabelText );
+  }
+
+  // background bar color
+  pbnBackgroundColor->setAllowOpacity( true );
+  pbnBackgroundColor->setColor( mDeco.mBackgroundColor );
+  pbnBackgroundColor->setContext( QStringLiteral( "gui" ) );
+  pbnBackgroundColor->setColorDialogTitle( tr( "Select Background Bar Color" ) );
+
+  // placement
+  cboPlacement->addItem( tr( "Top left" ), QgsDecorationItem::TopLeft );
```

Comment:
HIG Guideline is title case for combo box entries - "Top Left"

Reply:
@nyalldawson , I'll do this change in a separate commit, and apply proper title case across _all_ decoration dialogs. Good catch.